scripts/jointLklDM.C

```diff
@@ -910,6 +915,13 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
   for(Int_t imass=0;imass<nmass;imass++)
     cout << svSenVal[imass] << (imass<nmass-1? "," : "");
   cout << "};" << endl;
+  if(!channel.CompareTo("branon",TString::kIgnoreCase))
+    {
+      cout << "Double_t braneTensionLimit[nmass]  = {";
+      for(Int_t imass=0;imass<nmass;imass++)
+        cout << 0.001*TMath::Power((braneTensionVal[imass]*1.167*TMath::Power(10., -17.))/svLimVal[imass], 1./8.) << (imass<nmass-1? "," : "");
```

Comment:
Here I would suggest to use 1.1664 instead of 1.167 following the actual precision to which this constant is known (http://pdg.lbl.gov/2019/reviews/rpp2018-rev-phys-constants.pdf).

Reply:
Also I think it would be good to quote in a comment where the numbers comes form and what it represents.

Reply:
Also explains from which formula does the 0.001 (GeV to TeV), the 10^(-17) and the 1/8 comes from (just pointing to a reference where the formula is displayed is enough I think)

Reply:
Done!